spec/money_oxr/rates_store_spec.rb

```diff
@@ -103,6 +103,19 @@
       expect(subject.last_updated_at).not_to be nil
       expect(subject.last_updated_at.to_i).to be_within(10).of(Time.now.to_i)
     end
+
```

Comment:
I didn't separate it blocks with spaces here.  Can you make it consistent with the rest of the test?